messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_1_0.java

```diff
@@ -261,41 +261,43 @@
                                                 .addChild(
                                                         builder(SecurityRoleDefinition.INSTANCE)
                                                                 .addAttributes(
-                                                                        SecurityRoleDefinition.SEND,
-                                                                        SecurityRoleDefinition.CONSUME,
-                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.MANAGE,
                                                                         SecurityRoleDefinition.DELETE_NON_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.MANAGE)))
+                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CONSUME,
+                                                                        SecurityRoleDefinition.SEND
```

Comment:
The problem is https://issues.jboss.org/browse/WFCORE-1644.

All aspects of this PR that are driven by working around this bug need to be dropped. Perhaps that's everything but the IIOP bit?